pkg/openstack/loadbalancer.go

```diff
@@ -306,6 +310,73 @@ func (s *floatingSubnetSpec) Matcher(tag bool) (matcher, error) {
 	return match, nil
 }
 
+func initGetMaxOctaviaAPIVersion() func(client *gophercloud.ServiceClient) (*semver.Version, error) {
+	// Why all the timing cache hassle? The underlying OpenStack cloud might get updated at some point
+	// and we probably want to notice that.
```

Comment:
um... this is general update of openstack? if so I think worthy in utils instead of here

and the use case is openstack cloud got upgraded w/o notify CPO?

Reply:
Yeah, it's about the underlying OpenStack cloud. The K8s VMs are only a workload for OpenStack, so there's nothing that could notify CPO. Technically VMs will get restarted when compute nodes are upgraded but:
1. This might happen in the future, in bigger clouds computes might be upgraded one-by-one.
2. The VM CPO was running on could have been live-migrated, so never restarted.

Reply:
thanks for the info

that's the reason I am suggesting move to utils function instead of put here,,

Reply:
Oh, we even have a function for that in there already. Okay, I'll just use that.